pkg/util/validation/stack.go

```diff
@@ -37,14 +47,14 @@ func ValidateStackTags(tags map[apitype.StackTagName]string) error {
 	const maxTagValue = 256
 
 	for t, v := range tags {
-		if len(t) == 0 {
-			return errors.Errorf("invalid stack tag %q", t)
-		}
 		if len(t) > maxTagName {
-			return errors.Errorf("stack tag %q is too long (max length %d characters)", t, maxTagName)
+			return errors.Errorf("the stack tag name is too long (max length %d characters)", maxTagName)
```

Comment:
Any reason why you removed the case for `if len(t) == 0`? While perhaps a pathological error, `invalid stack tag name ""` is probably better in that case than `"stack tag names may only contain..."`. Since in the former case, we are showing the user what the invalid stack tag actually was.

Reply:
I don't remember, to be honest (Friday was a long time ago). I'll add it back in.